src/main/java/edu/rosehulman/covidtracer/service/PersonService.java

```diff
@@ -23,13 +23,20 @@
     private void showRepositoryInfo() {
         System.out.println("************PersonRepository autowired************  \n ->" + repository.getClass().getCanonicalName());
     }
-    
+
     public List<Person> getAllPersons(){
-        List<Person> result = repository.findAll();
-        if(!result.isEmpty()){
-            return result;
+        return repository.findAll();
+    }
+
+    public String getNameFromEmail(String email){
+        List<Person> people = repository.findByEmail(email);
+        System.out.println("We're doing this");
```

Comment:
Yayyy we are doing this!

Looks like a debugging statement, we can remove this.